bigquery/google/cloud/bigquery/external_config.py

```diff
@@ -41,49 +37,100 @@ class BigtableColumn(object):
     def __init__(self):
         self._properties = {}
 
-    encoding = _TypedApiResourceProperty(
-        'encoding', 'encoding', six.string_types)
-    """See
-    https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs#configuration.query.tableDefinitions.%28key%29.bigtableOptions.columnFamilies.columns.encoding
-    https://cloud.google.com/bigquery/docs/reference/rest/v2/tables#externalDataConfiguration.bigtableOptions.columnFamilies.columns.encoding
-    """
+    @property
+    def encoding(self):
+        """str: The encoding of the values when the type is not `STRING`
 
-    field_name = _TypedApiResourceProperty(
-        'field_name', 'fieldName', six.string_types)
-    """See
-    https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs#configuration.query.tableDefinitions.%28key%29.bigtableOptions.columnFamilies.columns.fieldName
-    https://cloud.google.com/bigquery/docs/reference/rest/v2/tables#externalDataConfiguration.bigtableOptions.columnFamilies.columns.fieldName
-    """
+        See
+        https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs#configuration.query.tableDefinitions.%28key%29.bigtableOptions.columnFamilies.columns.encoding
+        https://cloud.google.com/bigquery/docs/reference/rest/v2/tables#externalDataConfiguration.bigtableOptions.columnFamilies.columns.encoding
+        """
+        prop = self._properties.get('encoding')
```

Comment:
nit: couldn't this just be `return self._properties.get('encoding')` without the `prop =`?